osu.Game/Beatmaps/Formats/LegacyBeatmapEncoder.cs

```diff
@@ -0,0 +1,399 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using osu.Game.Audio;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Beatmaps.Legacy;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Types;
+
+namespace osu.Game.Beatmaps.Formats
+{
+    public class LegacyBeatmapEncoder
+    {
+        public const int LATEST_VERSION = 128;
```

Comment:
14 was the latest as per the wiki - is this a deliberate choice?

Reply:
#7116 introduced sliders with multiple segments. As each segment can have a different curve type, maps with such kind of pathes might be incompatible with 14.

Reply:
intentionally leaving some headroom in case we need to add more stable version increments (without conflicting with lazer).